webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -13,13 +13,15 @@
         <hc-user :user="currentUser" :trunc="35" />
         <ds-space />
         <ds-input model="title" class="post-title" placeholder="Title" name="title" autofocus />
+        <small class="smallTag">{{ form.title.length }}/{{ formSchema.title.max }}</small>
         <no-ssr>
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-18T09:06:52Z" title="Thursday, July 18th 2019, 11:06:52 am +02:00">Jul 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -10,13 +10,15 @@
           />
         </hc-teaser-image>
         <ds-input model="title" class="post-title" placeholder="Title" name="title" autofocus />
+        <small style="width:100%;position:relative;left:90%">{{ form.title.length }}/64</small>
         <no-ssr>
           <hc-editor
             :users="users"
             :hashtags="hashtags"
             :value="form.content"
             @input="updateEditorContent"
           />
+          <small style="width:100%;position:relative;left:90%">{{ form.contentLength }}/2000</small>
```


I would like more to have this not a hard code max length and been a speaking and centralised changeable const …

Ah, you have defined it here: `this.formSchema.content.max`